README.md

```diff
@@ -1,9 +1,47 @@
 # esprint [![npm version](https://img.shields.io/npm/v/esprint.svg?style=flat)](https://www.npmjs.com/package/esprint)
 
-esprint speeds up eslint by running the linting engine across multiple threads, and setting up a server daemon to cache the lint status of each file in memory. It uses a watcher to determine when files change, as to only re-lint what is necessary.
+esprint speeds up eslint by running the linting engine across multiple threads, and optionally sets up a server daemon to cache the lint status of each file in memory. It uses a watcher to determine when files change, as to only re-lint what is necessary.
 
 ## Usage
-TODO
+
+To run the esprint server, run the following command:
+
+```
+$ esprint start --workers=[num_workers] --port=[port_number]
```

Comment:
does just running `esprint` work? I think the command should be the same for starting and linting (to match flow and eslint). Also, we can move those args to `.esprintrc`